src/applications/personalization/profile-2/components/connected-apps/components/ConnectedApp.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import moment from 'moment';
+
+import PropTypes from 'prop-types';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+import recordEvent from 'platform/monitoring/record-event';
+
+import { AccountModal } from './AccountModal';
+
+class ConnectedApp extends React.Component {
+  constructor(props) {
```

Comment:
Not gonna tell you to make this a functional component, but I do dislike constructors :-D. You could simply do `state = { modalOpen: false, detailsOpen: false }` and remove the constructor entirely. More info https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599 (that post is basically impossible to read if your Mac is using Dark Mode, btw)

Reply:
Sounds great, will update! And thanks for the reference 👍